minimum-cluster-setup.md

```diff
@@ -361,7 +361,7 @@ You have PredictionIO in `~/pio` so edit ~/pio/conf/pio-env.sh to have these set
 	# Safe config that will work if you expand your cluster later
 	SPARK_HOME=/usr/local/spark
 	ES_CONF_DIR=/usr/local/elasticsearch
-	HADOOP_CONF_DIR=/usr/local/hadoop/etc/handoop
+	HADOOP_CONF_DIR=/usr/local/hadoop/etc/hadoop
```

Comment:
Yes, I will fix this, thanks you!  It is already fixed on the site docs which are going to replace these. Find the site markdown here: https://github.com/actionml/docs.actionml.com